src/components/NewProduct/NewProduct.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react';
+import ProductForm from '../ProductForm/ProductForm';
+import './NewProduct.scss';
+const NewProduct = () => {
+  const [newPhoto, setNewPhoto] = useState([]);
+  const [count, setCount] = useState(0);
+
+  useEffect(() => {
+    fetch('./data/New.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => [setNewPhoto(data)]);
+  }, []);
+
+  const testNext = () => {
+    if (count < newPhoto.length - 4 && count >= 0) {
+      setCount(count + 1);
```

Comment:
```suggestion
      setCount(count  => count + 1);
```
이전의 값을 가져와서 업데이트를 하는 경우에는 함수형 업데이트를 적용시켜주세요!

Reply:
감사합니다. 도현님 수정 완료 했습니다.